x/incentive/keeper/payout.go

```diff
@@ -167,6 +184,28 @@ func (k Keeper) SendTimeLockedCoinsToBaseAccount(ctx sdk.Context, senderModule s
 	return nil
 }
 
+// GetPeriodLength returns the length of the period based on the input blocktime and multiplier
+// note that pay dates are always the 1st or 15th of the month at 14:00UTC.
+func (k Keeper) GetPeriodLength(ctx sdk.Context, multiplier types.Multiplier) (int64, error) {
+
+	if multiplier.MonthsLockup == 0 {
+		return 0, nil
+	}
+	switch multiplier.Name {
+	case types.Small, types.Medium, types.Large:
```

Comment:
We can remove this switch statement since all cases execute the same logic and there's no default case specified.

Reply:
added a default 1d3a2e9